### DISCUSSION TITLE
Block Spotify Navigability with Samsung Screen Off in BMW ID6

### DISCUSSION THREAD

**Discussion by DagOriordan** [BimmerGestalt/AAIdrive#821] [Q&A]
Good morning, I would like to ask for information and solutions regarding the impossibility of navigating within Spotify when my Samsung Galaxy S23 Ultra has the screen off. Other users have complained about the same problem but, after a period in which everything was working, it has reappeared for a couple of days, even reinstalling the app from GooglePlay.
If I leave the phone with the screen on I can easily control Spotify from the iDrive while with the display off it doesn't allow me to do anything.
It freezes exactly after 1 minute!
What seems strange to me is that for even long periods everything works well, then suddenly it doesn't work anymore.
Thanks to anyone who can find a solution or give an explanation of this "strange phenomenon"!
I hope I was clear despite my Google English ;-)

**Comment by hufman** [comment]
This sounds exactly like the phone is causing the apps while the screen is off. Make sure both MyBMW and AAIdrive are allowed to run in the background.
Samsung uses heuristics to determine if an app should run in the background and learns for few days after installing it, and that's probably why you saw it working for a few days.

**Reply by DagOriordan** [reply]
Thanks for the reply, I'll check tonight or tomorrow. However I haven't seen it work just for days but for several months! Thanks again

**Comment by DagOriordan** [comment]
I've been doing some tests these days: I reinstalled the app and it seems to work correctly. I don't understand what could happen during use. A few months ago I had already set up that when Bluetooth was connected to the car, AAidrive, MyBMW and Spotify would start....so all the interesting apps were active. Strange world of technology!
Many thanks

**Reply by hufman** [reply]
I'm not sure how manually starting them influences things. Reinstalling probably resets whatever heuristics Samsung applies. I seem to remember that Samsung has two settings screens for applying battery optimizations, so make sure both are set to allow MyBMW to run in the background.

**Reply by DagOriordan** [reply]
Thanks, I didn't know about Samsung's "heuristic" mode. For now, after reinstallation, it works fine: when it gives problems, I will do a new installation of the software. The work done with AAIdrive is really nice and useful....it is acceptable to operate with uninstallation/installation every now and then.
Thanks for the assistance and for the work done....it is all much appreciated!
PS: Sorry for my "Google" English!
